packages/tools/devtools/devtools-core/src/ContainerDevtools.ts

```diff
@@ -288,7 +291,11 @@ export class ContainerDevtools implements IContainerDevtools, HasContainerKey {
 		[DisconnectContainer.MessageType]: async (untypedMessage) => {
 			const message = untypedMessage as DisconnectContainer.Message;
 			if (message.data.containerKey === this.containerKey) {
+				// Once disconnected, the devtools user will no longer be able to operate the container, it should be removed from the audience list.
+				const audience = this.container.audience as IAudienceOwner;
+				audience.removeMember(this.container.clientId as string);
```

Comment:
We shouldn't manually remove the member here. If the member is removed during `disconnect`, then that is an implementation detail of the container, not something we should be messing with in the devtools. Doing so could potentially cause problems if the implementation details in IContainer change.

Instead, it would probably be better to just post a message saying that the member was removed.